stdlib/SparseArrays/src/linalg.jl

```diff
@@ -529,15 +529,15 @@ end
 diff(a::SparseMatrixCSC; dims::Integer) = dims==1 ? sparse_diff1(a) : sparse_diff2(a)
 
 ## norm and rank
-vecnorm(A::SparseMatrixCSC, p::Real=2) = vecnorm(view(A.nzval, 1:nnz(A)), p)
+norm(A::SparseMatrixCSC, p::Real=2) = norm(view(A.nzval, 1:nnz(A)), p)
```

Comment:
do we actually need a `view` here?

Reply:
I think so… we don't want to make a copy of `A.nzval[1:nnz(A)]`.

Reply:
but why not use `A.nzval` directly?

Reply:
The `A.nzval` might be longer than `nnz(A)`.

Reply:
I see, thanks. Can that happen without manually messing with `A.nzval`?

Reply:
I thought it could happen with `.=` assignment to sparse matrices, but I don't remember clearly.

Reply:
Yes, the number of entries not matching the storage length can come about through e.g. broadcast.